### PR TITLE
Reusable infocard component#28

### PR DESCRIPTION
**Pull request by Marionwanyama23**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshot (optional)
image screenshot

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/home/src/lib/main/home/home-page.component.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <div class="home-page">
-  Elewa Education
-</div>
\ No newline at end of file
+  <elewa-website-elewa-info-card [cardInfo]="{title: 'Course design', description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Praesentium minima tempora magnam iusto.', icon: 'https://static.vecteezy.com/system/resources/previews/005/226/469/non_2x/rising-arrow-and-bar-graph-silhouette-icon-free-vector.jpg'}"> </elewa-website-elewa-info-card>
```
Comment (LemmyMwaura) [inline libs/features/pages/home/src/lib/main/home/home-page.component.html:2]: Custom Mock data should be stored in a data module. If not specified in the issue, then there's no need to push this changes

File: libs/elements/cards/src/lib/elewa-info-card/elewa-info-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { InfoCard } from '@elewa-website/models/schema/ui/cards';
+
+@Component({
+  selector: 'elewa-website-elewa-info-card',
+  templateUrl: './elewa-info-card.component.html',
+  styleUrls: ['./elewa-info-card.component.scss'],
+})
+export class ElewaInfoCardComponent {
+  @Input() cardInfo : InfoCard | undefined
+}
```
Comment (LemmyMwaura) [inline libs/elements/cards/src/lib/elewa-info-card/elewa-info-card.component.ts:11]: replace with `@Input() cardInfo!: InfoCard`

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Good job. Just clean up a few changes. Also update your PR description and remove the info card image from the codebase. Just Google how to add it to GitHub markdown.

**Comment by LemmyMwaura** [comment]
Kindly make the changes requested.

**Comment by Marionwanyama23** [comment]
I've made the necessary changes please review my code

**Comment by LemmyMwaura** [comment]
Closing as this has been completed in #48

**Pull request closed without merging**